packages/node_modules/@node-red/editor-client/src/js/ui/palette-editor.js

```diff
@@ -1120,8 +1133,9 @@ RED.palette.editor = (function() {
                     var descRow = $('<div class="red-ui-palette-module-meta"></div>').appendTo(headerRow);
                     $('<div>',{class:"red-ui-palette-module-description"}).text(entry.description).appendTo(descRow);
                     var metaRow = $('<div class="red-ui-palette-module-meta"></div>').appendTo(headerRow);
-                    $('<span class="red-ui-palette-module-version"><i class="fa fa-tag"></i> '+entry.version+'</span>').appendTo(metaRow);
-                    $('<span class="red-ui-palette-module-updated"><i class="fa fa-calendar"></i> '+formatUpdatedAt(entry.updated_at)+'</span>').appendTo(metaRow);
+                    $('<span class="red-ui-palette-module-version" title="Latest Version"><i class="fa fa-tag"></i> '+entry.version+'</span>').appendTo(metaRow);
+                    $('<span class="red-ui-palette-module-updated" title="Last Updated"><i class="fa fa-calendar"></i> '+formatUpdatedAt(entry.updated_at)+'</span>').appendTo(metaRow);
+                    $('<span class="red-ui-palette-module-downloads" title="Downloads - Last 7 Days"><i class="fa fa-download"></i> '+(new Intl.NumberFormat().format(entry.downloads.week))+'</span>').appendTo(metaRow);
```

Comment:
```suggestion
                    if (entry.downloads?.week !== undefined) {
                        $('<span class="red-ui-palette-module-downloads" title="Downloads - Last 7 Days"><i class="fa fa-download"></i> '+(new Intl.NumberFormat().format(entry.downloads.week))+'</span>').appendTo(metaRow);
                    }
```

Guard against `entry.downloads` not existing.